testing/integration/tx_relay_test.go

```diff
@@ -103,6 +103,7 @@ func TestTxRelay(t *testing.T) {
 						t.Fatal("Error payer is reciving")
 					}
 				}
+
```

Comment:
Please rollback this file (I know it's not important, but I don't want to spam the `git blame` log)

Reply:
so I'll revert back without the whitespace.